src/Components/Users/UserAdd.tsx

```diff
@@ -144,14 +144,20 @@ export const UserAdd = (props: UserProps) => {
 
   const [usernameExists, setUsernameExists] = useState<number>(0);
 
-  const checkUsername = async (username: string) => {
+  const check_username = async (username: string) => {
     setUsernameExists(userExistsEnums.checking);
-    const userDetails = await dispatchAction(getUserDetails(username), true);
-    setUsernameExists(
-      userDetails.status === 404
-        ? userExistsEnums.avaliable
-        : userExistsEnums.exists
+    const usernameCheck = await dispatchAction(
+      checkUsername({ username: username }),
+      false
```

Comment:
@siddnikh what does this `false` do?

Reply:
not sure. I changed it from `true` to this thinking it would silence the notification. I can't get anything from the docs either. I removed it and it still works the same. I'll push the changes.